lib/console/multi_buys/multi_buys.ex

```diff
@@ -25,4 +27,29 @@ defmodule Console.MultiBuys do
     |> MultiBuy.changeset(attrs)
     |> Repo.update()
   end
+
+  def get_all_multi_buy_associated_device_ids(id) do
+    device_ids =
+      get_associated_multi_buy_devices(id)
```

Comment:
can we just do a select of ids so we don't have to map over the results?

Reply:
good idea!

Reply:
though if it gives you `[%{ id: 'someid' }, %{ id: 'someotherid' }]`, you may have to iterate anyway?

Reply:
maybe  you could use `uniq_by` in that case?